packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -1835,7 +1836,7 @@ export class Global extends Op {
 
 	/**
 	 * Stores global field to query as string
-	 * @param args Expected arguments: [field] // Ex: ["GroupSize"]
+	 * @param args Expected arguments: [field] // Ex: [TxFieldEnum.GroupSize]
```

Comment:
What happens if we pass the string to a constructor? will it fail?

Reply:
Yes.

Reply:
I think we should allow it. Some users might be using it and this will be a breaking change.

Reply:
Sorry, are you talking the args param?
No, it won't break. The string is still allowed. We have an enum of fields in string so it will work.

Reply:
Yes, it still accepts string in the constructor.